tests/rbd/rbd_utils.py

```diff
@@ -1409,8 +1413,22 @@ def client_config_read_only(rbd, client_id, pool, **kw):
     osd_role = kw.get("osd_role", "allow *")
     cmd = f"ceph auth caps {client_id}"
     cmd += f" mon '{mon_role}' mds '{mds_role}'"
-    cmd += f" mgr '{mgr_role} pool={pool}"
-    if kw.get("namespace"):
```

Comment:
Why are we removing namespace argument here ?

Reply:
Keeping it straight to have any kind of caps for OSD, Mgr, Mon and Mds. So i removed namespace from here.